.github/workflows/ci.yml

```diff
@@ -26,19 +26,23 @@ jobs:
         with:
           repository: ${{ env.REPOSITORY }}
           github-token: ${{ secrets.LEON_REPOSITORY_DISPATCH }}
+          run-id: ${{ github.event.inputs.run-id }}
           name: build
           path: build
 
       - name: Disable Jekyll
         run: |
           touch build/.nojekyll
 
+      - name: Debug ls
```

Comment:
Still want this in?

Reply:
https://github.com/toitware/web-toitdocs-dev/pull/3
done.